go-app-chw.js

```diff
@@ -14,6 +15,39 @@ moment.parseTwoDigitYear = function (input) {
     return +input + (+input > 49 ? 1900 : 2000);
 };
 
+go.migration = {
+  make_migration_state: function (app, next_state) {
+    return (function(app) {
+      return function (name) {
+      // NOTE: only go through this if the migration flag is set
+      if(!app.im.config.migration_flag) {
+        return app.states.create(next_state);
```

Comment:
There was a bug in the previous PR (#230) as that returned a string here instead of the state with the name of whatever's stored in the `next_state` variable.